modules/site/src/app/examples/pagination/pagination-split-usage.component.scss

```diff
@@ -0,0 +1,63 @@
+@import "../../../ng2-material/core/style/variables";
+@import "../../../ng2-material/core/style/default-theme";
+@import "../../../ng2-material/components/whiteframe/whiteframe";
+
+pagination-split-usage {
```

Comment:
It's not the basic sample but a custom one. To make a book sample. The render is far from the default one, that's why there's some styles.